create_item_success.php

```diff
@@ -0,0 +1,4 @@
+
+
```

Comment:
`echo('<div class="text-center">Item successfully created!</div>');
  header("refresh:3;url=url_to_listing_page.php");`
  
  i think we can use something like this

Reply:
Should we redirect automatically?
I feel we should let the user chose to click on the view listing?

Reply:
But this is after user finish making item, right?

Flow is like: create item -> fill fields -> validate -> done -> go to listing?

Reply:
Yep, the last part -> go to listing, should that be automated via re-direct or only when the user actually clicks on "view listing" so that they get the time to read the success message ig?